src/elm/Page/Shop/Summary.elm

```diff
@@ -591,6 +594,19 @@ vatAmount price shared =
     (toFloat shared.remoteConfig.vat_percent / 100) * price
 
 
+iconForPaymentType : PaymentType -> Html Msg
+iconForPaymentType paymentType =
```

Comment:
Denne funksjonen er duplisert - kan legges i `PaymentType`. Trenger heller ikke å returnere `Html Msg`, `Html msg` bør fungere fint her.

Reply:
Jess. Tenkte på det, men var usikker på hvor jeg skulle legge det. Jeg så at ingen av de andre filene under `/Data` hadde bruk av `Html`, så jeg tenkte de skulle være rene data-klasser. Men det er altså greit med funksjoner under `/Data` som returnerer `Html msg`?

Reply:
Ja, hvis du lager datatyper som har enkle hjelpefunksjoner som f.eks. konvertere til String, Int, etc (samt parsing), og ting som dette hvor du egentlig bare er en velger for hvilket view som skal brukes, tenker jeg det er greit. Ordbruken med Data høres kanskje litt feil ut, men jeg vil heller ha én funksjon der enn flere funksjoner spredt over hele repoet. Hvis det blir mer en komponent, så kan man vurdere å splitte de to - en for datatypen og en for viewet.

Reply:
Done!